references/methods.csv

```diff
@@ -1,7 +1,15 @@
 Domain,Method,Description,Reference
+coastal,15s,water height and water height detided measured by coastal shores sensors every 15 seconds,
```

Comment:
@elidana 
"_Coastal waters are defined as the waters closest to the land. They include the waters of the Great Lakes,.." 
so i am happy ;)

Reply:
oh dear, it becomes very very long that way and talks more about the location rather than the method I think?